config/rapidez/sentry.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+return [
+    // Configuration as defined in the docs: https://docs.sentry.io/platforms/javascript/guides/vue/configuration/
+    'configuration' => [
+        'enabled' => true,
```

Comment:
Having an optional env variable for this would be nice. Default on; if needed it *could* be disabled through the env value.